test/server/cli-argument-parser-test.js

```diff
@@ -349,5 +350,45 @@ describe('CLI argument parser', function () {
                 expect(parser.opts.debugOnFail).to.be.ok;
             });
     });
+
+    it('Should has static CLI', function () {
+        var WARNING          = 'IMPORTANT: Please be sure what you want to change CLI if this test is failing!';
+        var EXPECTED_OPTIONS = [
+            { long: '--version', short: '-v' },
+            { long: '--list-browsers', short: '-b' },
+            { long: '--reporter', short: '-r' },
+            { long: '--screenshots', short: '-s' },
+            { long: '--screenshots-on-fails', short: '-S' },
+            { long: '--quarantine-mode', short: '-q' },
+            { long: '--debug-mode', short: '-d' },
+            { long: '--skip-js-errors', short: '-e' },
+            { long: '--test', short: '-t' },
+            { long: '--test-grep', short: '-T' },
+            { long: '--fixture', short: '-f' },
+            { long: '--fixture-grep', short: '-F' },
+            { long: '--app', short: '-a' },
+            { long: '--concurrency', short: '-c' },
+            { long: '--debug-on-fail' },
+            { long: '--app-init-delay' },
+            { long: '--selector-timeout' },
+            { long: '--assertion-timeout' },
+            { long: '--page-load-timeout' },
+            { long: '--speed' },
+            { long: '--ports' },
+            { long: '--hostname' },
+            { long: '--proxy' },
+            { long: '--qr-code' },
+            { long: '--color' },
+            { long: '--no-color' }
+        ];
+
+        var parser  = new CliArgumentParser('');
+        var options = parser.program.options;
+
+        expect(options.length).eql(EXPECTED_OPTIONS.length, WARNING);
+
+        for (var i = 0; i < EXPECTED_OPTIONS.length; i++)
```

Comment:
maybe
```
expect(options).to.includeDeepMembers(EXPECTED_OPTIONS)
```

Reply:
Looks good :)

Reply:
We found that it's not useful in this case.